tests/utils.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { PrismaTemplates } from '~/src'
+
+describe('getVercelDeployButtonUrl', () => {
```

Comment:
Let's get some coverage on this too now.

Reply:
I think you forgot to commit the snapshots though